src/pages/community/patterns/generate-an-api-key/index.mdx

```diff
@@ -95,7 +97,38 @@ A user may want to limit the access an application or service is granted. Restri
 <Row>
 <Column colLg={8}>
 
+![Many components can be implemented in the modal content space.](images/9.png)
+
+<Caption>
+  Many components can be implemented in the modal content space.
+</Caption>
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
 ![Example of a restricted API key generation](images/8.png)
 
+<Caption> Some examples of restricted API key generation</Caption>
+
+</Column>
+</Row>
+
+## Show the API key in context (optional)
+
+In some use-cases, it can be helpful to also show the user the API key in context, after the modal is closed. In this situation, the key should be displayed near the "Generate" button.  
+The user may also need the option to generate a new API key.
```

Comment:
Can you pls check the spacing between these two paragraphs? It may be that a soft return is in here and we'll need a hard return for the spacing to be correct. Thanks!